Microsoft.NET.Build.Containers/Image.cs

```diff
@@ -103,6 +117,27 @@ private static HashSet<Label> ReadLabelsFromConfig(JsonNode inputConfig)
         }
     }
 
+    private static Dictionary<string, string> ReadEnvVarsFromConfig(JsonNode inputConfig)
+    {
+        if (inputConfig is JsonObject config && config["Env"] is JsonArray envVarJson)
+        {
+            var envVars = new Dictionary<string, string>();
+            foreach (var entry in envVarJson)
+            {
+                if (entry is null)
+                    continue;
+                var val = entry.GetValue<string>().Split('=');
```

Comment:
probably want to do the thing here with `String.Split(splitter, max_num_of_segments)` so that we don't accidentally end up with more `=` splits than we anticipate.

Reply:
Thanks! I wish there was a "exactly_these_many" overload